os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -1650,9 +1654,13 @@ static int i2s_stop(struct i2s_dev_s *dev)
 	struct s5j_i2s_s *priv = (struct s5j_i2s_s *)dev;
 	irqstate_t flags;
 	struct s5j_buffer_s *bfcontainer;
+	volatile u32 reg;
 	DEBUGASSERT(priv);
 
 #ifdef I2S_HAVE_TX_P
+	modifyreg32(priv->base + S5J_I2S_CON, I2S_CR_TXDMACTIVE, 0);
```

Comment:
Why is this modified without disabling interrupts?

Reply:
Changed